Bindings~/python/pytest.ini

```diff
@@ -1,3 +1,4 @@
 [pytest]
 markers =
-    WebGLUnsupported: mark a test as a webtest.
\ No newline at end of file
+    WebGLUnsupported: mark a test as a webtest.
```

Comment:
I think this description is outdated, something like `mark a test as unsupported on WebGL.` would be more descriptive. The same for the other one.